### ISSUE TITLE
(no-trailing-whitespace) flags leading empty line as error

### ISSUE THREAD

**Issue by itanex** [palantir/tslint#4518]
### Bug Report

- __TSLint version__: 5.11.0
- __TypeScript version__: 2.9.2
- __Running TSLint via__:  CLI / (Visual Studio - does not show this error)

#### TypeScript code being linted
Multiple classes with a leading line that is empty. This line only contains line endings when analyzed (e.g. \r\n)
```ts

export class User {
    userName: string;
}
```

with `tslint.json` configuration:

```json
{
  "defaultSeverity": "error",
  "extends": "tslint:recommended",
  "jsRules": {},
  "rules": {
    "quotemark": [
      true,
      "single",
      "avoid-escape"
    ],
    "member-access": [
      true,
      "no-public"
    ],
    "object-literal-sort-keys": [
      false
    ],
    "object-literal-shorthand": [
      true,
      "never"
    ],
    "ordered-imports": [
      false
    ],
    "trailing-comma": [
      true,
      {
        "esSpecCompliant": true
      }
    ]
  },
  "rulesDirectory": [],
  "linterOptions": {
    "exclude": [
      "src/test.ts",
      "src/main.ts",
      "src/polyfills.ts"
    ]
  }
}
```

#### Actual behavior
Multiple results identical to the following

`ERROR: ClientApp/src/app/models/ActionItem.ts:1:1 - trailing whitespace`

#### Expected behavior
No results generated

**Comment by JoshuaKGoldberg** [comment]
@itanex thanks for the bug report! I tried reproducing this locally but couldn't. Is the project you're experiencing this with open source, or failing that, could you upload `ActionItem.ts` or another file it's happening on?

One guess would be that the file has some `\r\n` lines and some `\n`.

**Comment by itanex** [comment]
@JoshuaKGoldberg Yes the file line endings are windows line endings (\r\n).

I have uploaded the `ActionItem.ts` file along with the `tslint.json`. 

Please note that the `ActionItems.ts` file is generated through TypeWriter, so simply deleting the line is not a viable option, as this would need to be done anytime the file is regenerated.

**Comment by JoshuaKGoldberg** [comment]
This was a fun one to debug! The root issue is that `getLineRanges` from the `tsutils` library is reporting that the first line in the `ActionItem.ts` file has a `contentLength` of 1 instead of 0. Filed https://github.com/ajafff/tsutils/issues/94 to track.

Accepting PRs to fix in this library in the meantime, since even if tsutils releases a version with a fix, TSLint still accepts older versions of tsutils.
